### DISCUSSION TITLE
Holding the jump button

### DISCUSSION THREAD

**Discussion by Pokilokui** [Pinsplash/halflife2chaos#84] [General]
Is it possible to know how you made jumps repeat when you hold down the jump button ?

Thanks in advance

**Comment by Pinsplash** [comment]
in CGameMovement::CheckJumpButton(),

if ( mv->m_nOldButtons & IN_JUMP )
	return false;		// don't pogo stick

comment these lines out

**Reply by Pokilokui** [reply]
Thanks man
